packages/rest/src/validation/request-body.validator.ts

```diff
@@ -106,9 +105,13 @@ function validateValueAgainstJsonSchema(
   debug('Invalid request body: %s', util.inspect(ajv.errors));
 
   const error = RestHttpErrors.invalidRequestBody();
-  const pickedProperties = ['dataPath', 'keyword', 'message', 'params'];
   error.details = _.map(ajv.errors, e => {
-    return _.pick(e, pickedProperties);
+    return {
```

Comment:
Nitpic: I think this can be simplified as follows.

```ts
error.details = _.map(ajv.errors, e => {
  path: e.dataPath,
  // ...
  info: e.params,
});
```